packages/grafana-ui/src/options/builder/axis.tsx

```diff
@@ -213,3 +205,19 @@ export function addAxisWidth(builder: FieldConfigEditorBuilder<AxisConfig>) {
     showIf: (c) => c.axisPlacement !== AxisPlacement.Hidden,
   });
 }
+
+/** @internal */
+export function addAxisPlacement(
+  builder: FieldConfigEditorBuilder<AxisConfig>,
+  include = (placement: AxisPlacement) => true
```

Comment:
what do you think about `optionsInclude` to make it a bit more obvious what we are including?

Reply:
sure, or even `optionsFilter`